From f9a8c29a99bcbbf81f4c954a298990689b99bf35 Mon Sep 17 00:00:00 2001
From: Lukas Tribus <luky-37@hotmail.com>
Date: Thu, 26 Nov 2015 01:48:08 +0100
Subject: [PATCH 13/19] BUG/MINOR: lua: don't force-sslv3 LUA's SSL socket

Sander Klein reported an error messages about SSLv3 not
being supported on Debian 8, although he didn't force-sslv3.

Vincent Bernat tracked this down to the LUA initialization, which
actually does force-sslv3.

This patch removes force-sslv3 from the LUA initialization, so
the LUA SSL socket can actually use TLS and doesn't trigger
warnings when SSLv3 is not supported by libssl (such as in
Debian 8).

This should be backported to 1.6.
(cherry picked from commit d334a2c843c493c0bbd1bf12efa9f263b7f6cbea)
---
 src/hlua.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/hlua.c b/src/hlua.c
index 5ee2337..17e9b54 100644
--- a/src/hlua.c
+++ b/src/hlua.c
@@ -6475,7 +6475,6 @@ void hlua_init(void)
 		"ssl",
 		"verify",
 		"none",
-		"force-sslv3",
 		NULL
 	};
 #endif
-- 
2.4.10

